### PR TITLE
Kubeflow training service documentation, v1

### PR DESCRIPTION
**Pull request by yds05**
Kubeflow training service documentation, v1. 
Describe how to setup kubeflow training service environment, and set up NNI configuration for Kubeflow to launch experiment.

### CODE REVIEW

File: docs/KubeflowMode.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+**Run an Experiment on Kubeflow**
+===
+Now NNI supports running experiment on [Kubeflow](https://github.com/kubeflow/kubeflow), called kubeflow mode. Before starting to use NNI kubeflow mode, you should have a kubernetes cluster, either on-prem or [Azure Kubernetes Service(AKS)](https://azure.microsoft.com/en-us/services/kubernetes-service/), a Ubuntu machine on which [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) is installed and configured to connect to your kubernetes cluster. If you are not famaliar with kubernetes, [here](https://kubernetes.io/docs/tutorials/kubernetes-basics/) is a goot start. In kubeflow mode, your trial program will run as kubeflow job in kubernetes cluster.
```
Comment (chicm-ms) [inline docs/KubeflowMode.md]: typo: famaliar

File: docs/KubeflowMode.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+Note: You should explicitly set `trainingServicePlatform: kubeflow` in nni config yaml file if you want to start experiment in kubeflow mode. 
+
+Trial configuration in kubeflow mode have the following configuration keys:
+* codeDir
+    * code directory, where you put training code and config files
+* worker (required). This config section is used to configure tensorflow worker role
+    * replicas
+        * Required key. Should be positive number depends on how many replication your want to run for tensorflow worker role.
+    * command
+        * Required key. Command to launch your trial job, like ```python mnist.py```
```
Comment (chicm-ms) [inline docs/KubeflowMode.md]: python3  is better?

File: docs/KubeflowMode.md (hunk 1 above)
Comment (yds05) [reply docs/KubeflowMode.md]: yes

File: docs/KubeflowMode.md (hunk 2 above)
Comment (yds05) [reply docs/KubeflowMode.md]: yes

File: docs/KubeflowMode.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+# choice: true, false  
+useAnnotation: false
+tuner:
+  builtinTunerName: TPE
+  classArgs:
+    #choice: maximize, minimize
+    optimize_mode: maximize
+trial:
+  codeDir: ~/nni/examples/trials/mnist
+  Ps:
```
[the remark this answers is not in the document]
Comment (yds05) [reply docs/KubeflowMode.md]: yes. fixed

**Pull request merged**